.github/ct-install.yaml

```diff
@@ -5,6 +5,7 @@ chart-dirs:
 - charts/incubator
 - charts/stable
 excluded-charts:
+- charts/incubator/sdtd
```

Comment:
Why did you exclude the chart from the install CI? Can it not run without specific hardware or are certain values required that you do not wish to set by default?

Reply:
It needs to download the whole game before it is `Ready`, which is about > 10 GB.